.travis.yml

```diff
@@ -267,6 +267,13 @@ matrix:
     - $TRAVIS_BUILD_DIR/ci/travis_script_ruby.sh
   # Rust
   - language: rust
+    rust:
+      - stable
+      - nightly
+    matrix:
+      allow_failures:
+        - rust: stable
+      fast_finish: true
```

Comment:
We intentionally run both stable and nightly in a single CI step, see https://github.com/apache/arrow/pull/2477#issuecomment-416060848

Reply:
OK, let me think how to allow them in a single CI step _and_ allow failure in stable.

Reply:
- RUSTUP_TOOLCHAIN=stable $TRAVIS_BUILD_DIR/ci/travis_script_rust.sh || true
- RUSTUP_TOOLCHAIN=nightly $TRAVIS_BUILD_DIR/ci/travis_script_rust.sh

Reply:
Ha, I should have figured this out. Nevertheless #3034 is going to address this so I'll remove the related changes from this PR.